Bronte volatile commits

BronteCadogan opened the pull request:
Added Login ffeature and forum

BronteCadogan committed:
THIS BETTER WORK

BronteCadogan committed:
I COMMITED THE WRONG FILES

BronteCadogan committed:
Fixed issu ewith ignored

BronteCadogan committed:
Delete package.json

BronteCadogan committed:
Update App.js

BronteCadogan committed:
Add files via upload

BronteCadogan committed:
Add files via upload

loedge reviewed (changes requested):
A lot of your changes in App.js would overwrite existing changes and file structures in dev. I suggest pulling from dev into your branch and adapting your changes from there as to not overwrite existing data.

BronteCadogan committed:
forum included now

Note the html for the table and buttons did not transfer well from lab 4, I have no idea why
